Sources/ArcGISToolkit/Components/Offline/Utilities/URL+Offline.swift

```diff
@@ -40,4 +44,10 @@ extension URL {
         }
         return url
     }
+    
+    static func pendingMapInfoDirectory(
```

Comment:
```suggestion
    /// The path to the pending map info directory for a specific portal item.
    /// `Caches/PendingDownloads/<Portal Item ID>/`
    /// - Parameter portalItemID: The ID of the web map portal item.
    /// - Returns: A URL to the pending map info directory.
    static func pendingMapInfoDirectory(
```
Suggested documentation.